packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/extensions/error_model_translations.dart

```diff
@@ -53,3 +61,11 @@ extension ErrorServerGenericModelL10n on ErrorServerGenericModel {
   String translate(BuildContext context) =>
       message ?? context.l10n.error.server;
 }
+
+extension NoMailAppErrorModelL10n on NoMailAppErrorModel {
+  String translate(BuildContext context) => context.l10n.error.noMailApp;
+}
+
+extension InvalidUrlErrorModelL10n on InvalidUrlErrorModel {
+  String translate(BuildContext context) => context.l10n.error.invalidUrl;
+}
```

Comment:
These too could be added conditionally. Also, if this isn't being used elsewhere, you can mark those extensions as private.

Reply:
Moved to be part of the `feature_onboarding` brick and made them private